examples/paywall-tester/src/main/res/layout/activity_paywall_footer_view.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:tools="http://schemas.android.com/tools"
+        android:orientation="vertical"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        tools:context=".PaywallFooterViewActivity">
+
+    <LinearLayout
+            android:background="@android:color/holo_blue_light"
+            android:layout_weight="1"
+            android:layout_width="match_parent"
+            android:layout_height="0dp"
```

Comment:
Why is this 0dp?

Reply:
If you set the `layout_weight` parameter, you usually need to set one of the dimensions with 0dp so it can grow from that dimension. This is basically telling it to use all the available space after removing the size of other sibling views that don't use `layout_weight`.